pkg/k8shandler/deployment.go

```diff
@@ -48,6 +48,7 @@ func (node *deploymentNode) isDifferent(cfg *desiredNodeState) (bool, error) {
 	}
 
 	// TODO: Check if the Variables are the desired ones
+	return true, nil
```

Comment:
I haven't found an easy way to check if the env vars are different yet, this may become less hacky